repository.json

```diff
@@ -581,6 +581,23 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"details": "https://github.com/sublimelsp/lsp-rust-analyzer",
+			"name": "LSP-rust-analyzer",
+			"labels": [
+				"auto-complete",
+				"linting",
+				"formatting",
+				"code navigation",
+				"intellisense"
+			],
+			"releases": [
+				{
+					"sublime_text": ">=3092",
```

Comment:
Your package is only compatible with ST4 because you've used the `AbstractPlugin` API. This needs to be `>= 4070`